src/components/Questionnaire/QuestionnaireQuestions/Table/TableCommonComponents.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { memo } from 'react'
+import { Stack, Typography, MobileStepper, Button, TableRow, TableCell, CircularProgress } from '@common/components'
+import { KeyboardArrowLeft, KeyboardArrowRight } from '@common/icons'
+import { useTranslation } from 'react-i18next'
+
+type ButtonStackProps = {
+  activeStep: number
+  handleNext: () => void
+  handleBack: () => void
+  steps: number
+  idType: string
+  disabled: boolean
+}
+
+type SendButtonProps = {
+  t: (key: string) => string
+  handleSend: () => void
+  isNextDisabled: boolean
+  isValid: boolean
+  idType: string
+  isSending: boolean
+  sendSuccess: boolean
+}
+
+/**
+ * @param {string} question - The question to be displayed in this row
```

Comment:
{string} gibts net in typedoc. Kannst das einfach entfernen der erkennt automatisch